content/W2015/W15-event-boardgames.md

```diff
@@ -13,6 +13,8 @@ some new faces.
 
 Hope to see you there!
 
+Our Big CSters program helps connect ciswomen, transwomen, and non-binary students with other students in CS and related fields, through social events geared towards mentorship.
```

Comment:
I used slightly different wording in the FAQ: https://github.com/wics-uw/website/pull/46/files#diff-5bb30c3825633713c9eb52a303b2b228R11

We should definitely be consistent. Would like opinions on which one we should change :)

Also, this needs to be 80-char justified, like the rest of the article. I think we should include an "About Big CSters:" text in bold or something to differentiate it from the rest of the text as well!